twig/report/page/appendix_table.html.twig

```diff
@@ -11,11 +11,13 @@
       </thead>
       <tbody>
         {% for result in assessment.results %}
```

Comment:
How about this instead:
` {% for result in assessment.results|filter(r => r.getType != 'notice')  %}`

Reply:
Works for me. PR updated.